src/sensors/drivers/adafruit_dht22.cpp

```diff
@@ -1,5 +1,16 @@
 #include "sensors/drivers/adafruit_dht22.h"
 #include "system/logs.h" // for debug() and notify()
+#include "system/hardware.h" // for pin names
+
+short GPIO_PINS[7] = {
+    GPIO_PIN_1,
+    GPIO_PIN_2,
+    GPIO_PIN_3,
+    GPIO_PIN_4,
+    GPIO_PIN_5,
+    GPIO_PIN_6,
+    GPIO_PIN_7
+};
```

Comment:
They are globally available, this is just an int array within the dht22 driver for now, same as the generic_analog driver having an int array for the internal analog pins. To pair the int with a #define in hardware.h
I do think it would be better to have all of our pins in one place with one naming scheme for them though. I'm not sure it's helpful to have an extra name for them outside of their original pin names.  The pin names could be printed on the board too.  It gets weird when the pins are part of sensor_ports.  So, it depends on whether they'll be accessed through a port or directly by an exposed pin/pad.

Reply:
what connector is typically going to be used when a GPIO device is being plugged in?  probably needs power and ground too, right?

Reply:
Right now we're using 5v power off of an exADC port, and a pin from the STM32 headers. So, it depends on how much power the sensor needs. In our case we wouldn't be able to use the internal ADC ports for the DHT22 because it needs more power. So it'll be case by case basis unless we can figure out a good standard port/power to use where it's no longer an issue.

Reply:
Yeah, lots of different kinds of configurations could pop up.  I'm not what the best generic approach could be.  Making ports available for all the different kinds of configs, setting up the board with jumpers that can be moved around to change the configuration, or just letting the user wire into whatever pins they want to.